pkg/harvester/validators/volume.js

```diff
@@ -1,5 +1,11 @@
 export function volumeSize(size, getters, errors, validatorArgs, displayKey, value) {
-  if (!/^([1-9][0-9]{0,8})[a-zA-Z]+$/.test(size)) {
+  if (!size) {
+    const key = getters['i18n/t']('harvester.volume.size');
+
+    errors.push(getters['i18n/t']('validation.required', { key }));
+  }
+
+  if (size && !/^([0-9][0-9]{0,8})[a-zA-Z]+$/.test(size)) {
```

Comment:
I also modify the regex : `/^([0-9][0-9]{0,8})[a-zA-Z]+$/` to allow user input 0 size. The API will return clear empty message for user.

<img width="1488" alt="Screenshot 2024-05-12 at 10 55 41 PM" src="https://github.com/harvester/dashboard/assets/5744158/04c736d8-6694-4174-9014-15cf069e0fcc">